contrib/packages/deb/ubuntu-noble/debian/control

```diff
@@ -0,0 +1,193 @@
+Source: tigervnc
+Section: x11
+Priority: optional
+Maintainer: Brian P. Hinz <bphinz@users.sourceforge.net>
+Standards-Version: 3.8.4
+Build-Depends:
+# TigerVNC dependencies
```

Comment:
I see you did some cleanups in this list. Perhaps best to do in the other variants as well to make things easy to keep in sync?

On that note, gcrypt is not that commonly used any more. Are we sure Ubuntu still uses that for Xorg?

Reply:
libgcrypt is still listed as a build requirement in Ubuntu's xorg-server package